src/service/url-replacements-impl.js

```diff
@@ -451,7 +451,7 @@ export class UrlReplacements {
       return Promise.resolve();
     } else if (metric < 0) {
       // Metric is not yet available. Retry after a delay.
-      return loadPromise(this.win_).then(() => {
+      return loadPromise(this.win_.document).then(() => {
```

Comment:
That makes sense. We should definitely switch to `onDocumentReady` then. the second use case in `getTimingData` is not super obvious to me though yet. I'm not sure what's the goal there.